lib/rust/ensogl/core/Cargo.toml

```
@@ -38,6 +38,7 @@ nalgebra = { version = "0.26.1" }
 num_enum = { version = "0.5.1" }
 num-traits = { version = "0.2" }
 paste = { version = "1.0.5" }
+profiling = { path = "../../../../lib/rust/profiling" }
```

```suggestion
profiling = { path = "../../profiling" }
```
This is the convention we follow so far.